src/services/rollouts.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-useless-catch */
+/* eslint-disable no-undef */
+const httpStatusCode = require('@generics/http-status')
+const responses = require('@helpers/responses')
+const common = require('@constants/common')
+const utils = require('@generics/utils')
+const rolloutQueries = require('@database/queries/rollouts')
+const resourceService = require('@services/resource')
+const resourceQueries = require('@database/queries/resources')
+module.exports = class RolloutsHelper {
+	/**
+	 * Rollout create
+	 * @method
+	 * @name create
+	 * @param {Object} req - request data.
+	 * @returns {JSON} - project id
+	 */
+	static async create(bodyData, loggedInUserId, orgId) {
+		try {
+			//validate the title length
+			const isTitleInvalid = utils.validateTitle(bodyData.title)
+			if (isTitleInvalid) {
+				return responses.failureResponse({
+					message: 'CHARACTER_LIMIT_EXCEED',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			//validation for start date and end date
+			if (bodyData.start_date && bodyData.end_date) {
```

Comment:
This also can we do it in validator? So that these basic checks are validated before we reach service file @priyanka-TL